### PR TITLE
Added Jenkins notes

### PR DESCRIPTION
**Pull request by matq007**
Adding notes about in-house Jenkins cluster. Will be updated a few more times.

### CODE REVIEW

**Comment by matq007** [comment]
Ok, I think we are ready for a review. We can update the docs continuously.

**Comment by alneberg** [comment]
Great that you were able to document the setup so well! 

However, it would be nice if the README had a bit more of an introduction. I think it should include:
 - What this is
 - That its currently setup within SciLifeLab but it's not accessible outside of the network
 - That it currently runs for every nf-core pipeline which has a jenkins file.

I'm not the most suitable one to tell whether this is the best location for this docs though. Maybe @apeltzer or @sven1103 might be better off answering that question?

**Comment by apeltzer** [comment]
My comments were not added from my mobile phone, so I just follow up on what @alneberg just posted. I think this is already close to being finished, but maybe it makes sense to have this very useful part of the documentation on the actual webpage (https://github.com/nf-core/nf-co.re) as a new subsection

Bildschirmfoto 2019-05-16 um 16 21 34

We could have a separate section there, saying Pipeline Testing - then add both the TravisCI testing and your Jenkins docs. 

Adding the stuff that @alneberg mentioned would be nice then too, so that people don't think they can access the Jenkins server themselves directly.

File: docs/jenkins/README.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+sh install.sh
+docker-compose up -d
+```
+
+To rebuild the stack run:
+
+```bash
+docker-compose down
+docker-compose build
+docker-compose up --force-recreate -d
```
Comment (MaxUlysse) [inline docs/jenkins/README.md]: Can't we use?
`docker-compose up --force-recreate --build -d`

Comment (apeltzer) [reply docs/jenkins/README.md]: Indeed we should be able to do that!

Comment (MaxUlysse) [reply docs/jenkins/README.md]: How did you do last time to propose changes?

Comment (matq007) [reply docs/jenkins/README.md]: Didn't have time to get to it. I will work on it this week.

**Comment by apeltzer** [comment]
@matq007 Any update on this ?

**Comment by matq007** [comment]
> @matq007 Any update on this ?

Ah, didn't get to this yet. Will post update ASAP.

File: docs/jenkins/README.md (hunk 1 above)
Comment (ewels) [reply docs/jenkins/README.md]: @MaxUlysse:
image

Comment (MaxUlysse) [reply docs/jenkins/README.md]: Thanks a lot, I did not notice this icon

Comment (MaxUlysse) [reply docs/jenkins/README.md]:
```suggestion
```bash
docker-compose up --force-recreate -d
```

Comment (MaxUlysse) [reply docs/jenkins/README.md]:
```suggestion
docker-compose up --force-recreate -d
```

**Comment by matq007** [comment]
Moving PR to nf-co.re repo. x-ref: https://github.com/nf-core/nf-co.re/pull/154

**Pull request closed without merging**